src/instrumenter.js

```diff
@@ -158,12 +159,12 @@ function getVersion (moduleBaseDir) {
 }
 
 function filename (plugin) {
-  return [plugin.name, plugin.file].filter(val => val).join('/')
+  return path.join.apply(path, [plugin.name, plugin.file].filter(val => val))
 }
 
 function exists (basedir, file) {
   try {
-    require.resolve(`${basedir}/${file}`)
+    require.resolve(path.join(basedir, file))
```

Comment:
I don't think this is necessary since `require` uses the module syntax and not regular paths, meaning it should always be forward slashes. See https://github.com/nodejs/node/issues/6049#issuecomment-205778576